app/src/main/java/com/bucketlist/ui/main/MainFragment.kt

```diff
@@ -9,26 +9,18 @@ import android.view.ViewGroup
 import com.bucketlist.R
 
 class MainFragment : Fragment() {
-
-    companion object {
-        fun newInstance() = MainFragment()
-    }
-
     private lateinit var viewModel: MainViewModel
-
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
         return inflater.inflate(R.layout.main_fragment, container, false)
     }
-
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
-        // TODO: Use the ViewModel
     }
-
-
-
+    companion object {
+        fun newInstance() = MainFragment()
+    }
```

Comment:
I Changed the location of the Companion object based on the order of classes that need to be followed.
Check out: https://kotlinlang.org/docs/reference/coding-conventions.html